### ISSUE TITLE
win32 buildfarm failure on nightly

### ISSUE THREAD

**Issue by sklam** [numba/numba#8972]
Since merging of https://github.com/numba/numba/pull/8633

All win32 buildfarm builds are failing with:

```

======================================================================
FAIL: test_guvectorize_dynamic (numba.tests.doc_examples.test_examples.DocsExamplesTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\ci\32\miniconda3\envs\testenv_049a9f14\lib\site-packages\numba\tests\doc_examples\test_examples.py", line 566, in test_guvectorize_dynamic
    self.assertEqual(g.types, correct)
AssertionError: Lists differ: ['ql->q'] != ['qq->q']

First differing element 0:
'ql->q'
'qq->q'

- ['ql->q']
?    ^

+ ['qq->q']
?    ^


======================================================================
FAIL: test_vectorize_dynamic (numba.tests.doc_examples.test_examples.DocsExamplesTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\ci\32\miniconda3\envs\testenv_049a9f14\lib\site-packages\numba\tests\doc_examples\test_examples.py", line 488, in test_vectorize_dynamic
    self.assertEqual(f.types, correct)
AssertionError: Lists differ: ['ll->l'] != ['ll->q']

First differing element 0:
'll->l'
'll->q'

- ['ll->l']
?       ^

+ ['ll->q']
?       ^


----------------------------------------------------------------------
Ran 11644 tests in 4031.736s
```

**Comment by gmarkall** [comment]
By win32 do you mean 32- or 64-bit Windows?

**Comment by sklam** [comment]
just 32-bit

**Comment by esc** [comment]
Closing this as unsupported. With 0.57.1 released and 0.58.0dev0 in progress, we have removed the `win32` specific configuration for testing Numba on the buildfarm.

**Comment by esc** [comment]
ref: #9041
